### PR TITLE
Minor wording changes

### PR DESCRIPTION
**Pull request by SvenAelterman**
"pool in a SQL Database" gives the impression that the pool is a child of a database, which is incorrect.
Also removed unnecessary line breaks.
Added PremiumRS as an Edition value and re-arranged acceptable values in logical order.
Set -Tags example with code formatting and changed a tag name to include a space, to illustrate the possibility.

## Description

---

This checklist is used to make sure that common guidelines for a pull request are followed. You can find a more complete discussion of PowerShell cmdlet best practices [here](https://msdn.microsoft.com/en-us/library/dd878270(v=vs.85).aspx).

- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md).**
- [x] **If changes were made to any cmdlet, the XML help was regenerated using the [platyPSHelp module](https://github.com/Azure/azure-powershell/blob/preview/documentation/help-generation.md).**
- [x] **If any large changes are made to a service, they are reflected in the respective [change log](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#updating-the-change-log).**

### [General Guidelines](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#general-guidelines)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/preview/documentation/cleaning-up-commits.md).
- [x] The pull request does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/preview/documentation/breaking-changes/breaking-changes-definition.md) (unless a major version change occurs in the assembly and module).

### [Testing Guidelines](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#testing-guidelines)
- [x] Pull request includes test coverage for the included changes.
- [x] PowerShell scripts used in tests should do any necessary setup as part of the test or suite setup, and should not use hard-coded values for locations or existing resources.

### [Cmdlet Signature Guidelines](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#cmdlet-signature-guidelines)
- [x] New cmdlets that make changes or have side effects should implement `ShouldProcess` and have `SupportShouldProcess=true` specified in the cmdlet attribute. You can find more information on `ShouldProcess` [here](https://github.com/Azure/azure-powershell/wiki/PowerShell-Cmdlet-Design-Guidelines#supportsshouldprocess).
- [x] Cmdlet specifies `OutputType` attribute if any output is produced - if the cmdlet produces no output, it should implement a `PassThru` parameter.

### [Cmdlet Parameter Guidelines](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#cmdlet-parameter-guidelines)
- [x] Parameter types should not expose types from the management library - complex parameter types should be defined in the module.
- [x] Complex parameter types are discouraged - a parameter type should be simple types as often as possible. If complex types are used, they should be shallow and easily creatable from a constructor or another cmdlet.
- [x] Cmdlet parameter sets should be mutually exclusive - each parameter set must have at least one mandatory parameter not in other parameter sets.

### CODE REVIEW

File: src/ResourceManager/Sql/Commands.Sql/help/Set-AzureRmSqlElasticPool.md [hunk 1]
```diff
@@ -230,7 +228,7 @@ Accept wildcard characters: False
 Specifies a dictionary of Key-value pairs that this cmdlet associates with the elastic pool in the
 form of a hash table. For example:
 
-@{key0="value0";key1=$null;key2="value2"}
+`@{key0="value0";"key 1"=$null;key2="value2"}`
```
Comment (cormacpayne) [inline src/ResourceManager/Sql/Commands.Sql/help/Set-AzureRmSqlElasticPool.md]: @SvenAelterman any reason for this change to the sample hashtable?

File: src/ResourceManager/Sql/Commands.Sql/help/Set-AzureRmSqlElasticPool.md [hunk 2]
```diff
@@ -278,14 +276,10 @@ Accept wildcard characters: False
 ### CommonParameters
 This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see about_CommonParameters (http://go.microsoft.com/fwlink/?LinkID=113216).
 
-## INPUTS
-
 ## OUTPUTS
 
 ### Microsoft.Azure.Commands.Sql.ElasticPool.Model.AzureSqlElasticPoolModel
 
-## NOTES
```
Comment (cormacpayne) [inline src/ResourceManager/Sql/Commands.Sql/help/Set-AzureRmSqlElasticPool.md]: @SvenAelterman please add back the `## INPUTS` and `## NOTES` headers that were removed

**Comment by SvenAelterman** [comment]
@cormacpayne It's not particularly big deal, but just an illustration of how a tag name could have a space in it...

File: src/ResourceManager/Sql/Commands.Sql/help/Set-AzureRmSqlElasticPool.md (hunk 1 above)
Comment (SvenAelterman) [reply src/ResourceManager/Sql/Commands.Sql/help/Set-AzureRmSqlElasticPool.md]: Just indicating that a name with spaces is possible if it's enclosed in quotes.

**Comment by SvenAelterman** [comment]
@cormacpayne Re-added level 2 headings as requested.
I suppose I don't understand the purpose of the empty headings?

**Comment by cormacpayne** [comment]
@SvenAelterman we have tooling in place for markdown validation, and that tooling depends on the existence of these headers. The headers are also generated by the `platyPS` module we use to generate markdown help from the binary cmdlets, so they would be added back whenever someone updates the Sql cmdlets in the future.

**Comment by cormacpayne** [comment]
@azuresdkci add to whitelist

**Pull request merged**